floodwatch/src/js/components/Options.js

```diff
@@ -0,0 +1,114 @@
+// @flow
+
+import React, {Component} from 'react';
+import { MenuItem, ButtonToolbar, DropdownButton } from 'react-bootstrap';
+import Filters from '../../stubbed_data/filter_response.json';
+import {CustomFilter} from './CustomFilter'
+import {getCategoryKey, getCategoryOfUserVal, shouldPresetBeDisabled, shouldCustomBeDisabled} from './FindInDemographics'
+import type {Filter, Preset, PresetsAndFilters, FilterJSON} from './filtertypes.js'
+import type {PersonResponse} from '../api/types';
+
+
+//-------------------------------------------------------------------------------------------------------------------
+export class RegularOptions extends Component {
+  render() {
+    return (
+      <p>Ads seen by {this.props.currentSentence}.</p>
+    )
+  }
+}
+
+//-------------------------------------------------------------------------------------------------------------------
+type CustomOptionsProps = {
+  currentSelection: Array<Filter>,
+  handleFilterClick: Function,
```

Comment:
Having just traced the origin of this function, it might be worth annotating the detailed type here because it goes a looong way (and changes names) to get here.